server/src/cache/mod.rs

```diff
@@ -57,10 +55,12 @@ where
     fn sleep_ms(&self) -> u64;
 
     /// Starts a loop to consume data from the Redis queue
+    #[cfg(not(tarpaulin_include))]
+    // no_coverage: loop, needs redis backend to test
```

Comment:
Your script to test the no_coverage reasons requires a release tag. I've used the tag `Rnever` for code that can never be tested

Reply:
Fixed, see new release checklist generated